Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java

```diff
@@ -277,24 +280,31 @@ private void setEditable(){
 	
 	/**
 	 * Sets the editable state of the entire UI.
-	 * @param editable Whether the claim is editable or not.
 	 */
 	private void setDeletable() {
-		if (editable) {
-			getListView().setOnItemLongClickListener(
-				new LongClickDeleteListener(this, new LongClickDeleteListener.OnDeleteListener() {
-					@Override
-					public void onDelete(int position) {
-						if (getListView().getItemAtPosition(position) == null) {
-							return;
-						}
-						longPressedItemPosition = itemPositionForListViewPosition(position);
-						ExpenseClaimDetailController.DetailItem.ItemType type = controller.getItemType(longPressedItemPosition);
-						startDeleteAlertDialog(type);
+		getListView().setOnItemLongClickListener(
```

Comment:
This should eventually be refactored to be part of LongClickDeleteListener but you can leave it this way for now if you want.